### DISCUSSION TITLE
Inline match URLs are not supported

### DISCUSSION THREAD

**Discussion by RickJFinn** [synthetichealth/synthea#1526] [Q&A]
Hello there, 
I'm attempting to upload Synthea data to the HAPI FHIR test server (http://hapi.fhir.org/baseR4) and getting the following error:
"HAPI-2282: Inline match URLs are not supported on this server. Cannot process reference: \"Organization?identifier=https://github.com/synthetichealth/synthea|62b55e5b-b58f-354d-8f2d-f2a330bc619e\""

Specifically, I'm trying to post a transaction bundle. I'm able to successfully post the hospitalInformation* and practitionerInformation* bundles, but when I get to the patient bundle, I get the above error. 
Any help would be greatly appreciated!

**Comment by jawalonoski** [comment]
Not sure what is going on there.

Querying https://hapi.fhir.org/baseR4/Organization?identifier=https://github.com/synthetichealth/synthea|62b55e5b-b58f-354d-8f2d-f2a330bc619e returns a result... so that resource exists.

If I locally run the latest code on the `master` branch of the `hapi-fhir-jpaserver-starter`, I can successfully post both the `hospitalInformation` and `practitionerInformation` file and then load the patient bundles...

Maybe the demo version of HAPI is running with a particular configuration setting?
